packages/platform-sdk/test/integrationTests/test-utils.ts

```diff
@@ -0,0 +1,30 @@
+import { ClientBuilder } from '@commercetools/sdk-client-v2/src'
```

Comment:
This should fix the `invalid token error`

```suggestion
import { ClientBuilder } from '@commercetools/sdk-client-v2'
```